dowhy/causal_estimators/gformula_estimator.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+from sklearn import linear_model
+import pandas as pd
+from dowhy.causal_estimator import CausalEstimate
+from dowhy.causal_estimator import CausalEstimator
+import logging
+
+
+class GFormulaEstimator(CausalEstimator):
```

Comment:
currently, this looks very similar to the LinearRegressionEstimator class. consider merging with it and adding the  do method there?